src/robot/running/model.py

```diff
@@ -175,6 +176,30 @@ def run(self, context, run=True, templated=False):
                 raise ReturnFromKeyword(self.values)
 
 
+@Body.register
+class Continue(model.Continue):
+    __slots__ = ['lineno']
+
+    def __init__(self, values=(), parent=None, lineno=None):
+        model.Return.__init__(self)
```

Comment:
Should be `model.Continue`, not `model.Return` here! Even better would be using `super()` that we have now started to use when we don't need to care about Python 2 (`super()` syntax was different with it). Also need to pass `parent` to the parent class. In other words, this should be `super().__init__(parent)`.